src/mango/src/mango_cursor_view.erl

```diff
@@ -94,18 +93,20 @@ maybe_replace_max_json(?MAX_STR) ->
 
 maybe_replace_max_json([H | T] = EndKey) when is_list(EndKey) ->
     H1 = if H == ?MAX_JSON_OBJ -> <<"<MAX>">>;
-            true -> H
+        true -> H
     end,
     [H1 | maybe_replace_max_json(T)];
 
 maybe_replace_max_json(EndKey) ->
     EndKey.
 
 
-base_args(#cursor{index = Idx, selector = Selector} = Cursor) ->
+base_args(#cursor{index = Idx} = Cursor) ->
     {StartKey, EndKey} = case Cursor#cursor.ranges of
         [empty] ->
             {null, null};
+        [] ->
+            {undefined, undefined};
```

Comment:
That's not correct though, right? A string is smaller than an empty object. Returning docids in response to that would break the violate the condition as specified in the selector. If memory serves, the hack for doing a complete scan was `{$gt: null}` because null sorts before everything else.